frontend/mr-admin-flate/src/components/avtaler/SlettAvtaleModal.tsx

```diff
@@ -71,18 +71,20 @@ const SlettAvtaleModal = ({
   function modalInnhold(avtale?: Avtale) {
     return (
       <>
-        {avtale?.opphav === Opphav.ARENA ? (
+        {avtaleFraArena ? (
           <p>Avtalen {avtale?.navn} kommer fra Arena og kan ikke slettes her</p>
         ) : mutation?.isError ? (
           <p>{(mutation.error as ApiError).body}</p>
         ) : (
           <p>Du kan ikke angre denne handlingen</p>
         )}
         <div className={styles.knapperad}>
-          {avtale?.opphav === Opphav.ARENA ? null : mutation?.isError ? (
-            <Button variant="primary" onClick={handleRedigerAvtale}>
-              Rediger avtale
-            </Button>
+          {avtaleFraArena ? null : mutation?.isError ? (
+            <Lenke to={`/avtaler/skjema?avtaleId=${avtale?.id}`}>
+              <Button variant="primary" onClick={handleRedigerAvtale}>
+                Rediger avtale
+              </Button>
+            </Lenke>
```

Comment:
Dette er ikke gyldig html. Du kan ikke ha `button` inne i en `a`-tag. 

Ref: https://stackoverflow.com/questions/6393827/can-i-nest-a-button-element-inside-an-a-using-html5

Du må nok style lenken til å se ut som en knapp istedenfor.

Reply:
Ah, da lager jeg en egen komponent for det